mkosi/__init__.py

```diff
@@ -2157,41 +2157,68 @@ def calculate_signature(context: Context) -> None:
     if not context.config.sign or not context.config.checksum:
         return
 
-    cmdline: list[PathString] = ["gpg", "--detach-sign", "--pinentry-mode", "loopback"]
+    tool = context.config.openpgp_tool
```

Comment:
Can we name this `pgptool` or something like that? Tools trees are most often referred to as `tools` and with only one letter difference, this might be confusing on such a long stretch.